### PR TITLE
Add contributors.md 

### LINKED ISSUE #18: Add Contributors File
### Problem
Nail down contribution guidelines and add contribute file.

### Resources
[Contributing file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors)
[CONTRIBUTING.md example](https://github.com/github/docs/blob/main/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by cbhernan**
### What does this PR do?
This PR adds contributing guidelines.

### Which issue(s) is/are related to this PR?
Closes #18

### CODE REVIEW

File: CONTRIBUTING.md
Comment (jaylenw) [inline CONTRIBUTING.md:22]: Are you okay with them opening a PR without input from you and your team first or would you rather them open an issue or refer to an existing issue regarding the problem for discussion before opening a PR? Personally, I would want potential contributors to have a small discussion and possibly a green light from the repo maintainers of what should be implemented prior to the work of a PR.

File: CONTRIBUTING.md
Comment (jaylenw) [inline CONTRIBUTING.md]: Ah ok, this addresses my previous comment. Perhaps have the "Reporting bugs or requesting features" section before the "Submitting Changes". Let's also capitalize the first letter of each word except "or".

**Review by jaylenw — CHANGES_REQUESTED** [review]
After reviewing best practices on creating a `CONTRIBUTING.md` file, what you have here is pretty solid. One of the items I referenced for my review was this here https://github.com/github/opensource.guide/blob/main/CONTRIBUTING.md.

I recommend noting in an Issue to update this file to mention the code of conduct file so users may know which type of behavior is and what is not expected from them. For this you may reference https://github.com/github/opensource.guide/blob/main/CODE_OF_CONDUCT.md and https://www.contributor-covenant.org/.

File: CONTRIBUTING.md
Comment (technima) [reply CONTRIBUTING.md]: agreed

**Comment by cbhernan** [comment]
Thanks for the feedback @jaylenw - I've gone ahead and made the updates.

I've added a sentence under `How to Contribute` that touches on being given a green light in order to contribute.

Let me know your thoughts.

cc: @technima

File: CONTRIBUTING.md
Comment (jaylenw) [inline CONTRIBUTING.md]: What are you thoughts in combining the `## How to Contribute` section and the `## Reporting Bugs or Requesting Features` section together? Have the `## Reporting Bugs or Requesting Features` section be a sub-heading.

Then, perhaps we modify this sentence here `All work submitted must be accompanied by a related Issue. Before starting work, ensure that the Maintainers of the project have given you a green light to go ahead and start work; this will ensure that no duplicate work is being done and will allow the Maintainers to share any things to keep in mind for your update.` to `All work submitted must be accompanied by a GitHub Issue. After creating a GitHub Issue and providing the required information mentioned below, please wait for feedback from the repository maintainers before proceeding on the implementation. This will ensure that the collaborative and development process is efficient and coordinated to make sure your time and efforts are used wisely.`

Then follow up with the content that exists in the `## Reporting Bugs or Requesting Features`  section but add an extra `#` to make it a subdeading.

--------------

Let me know your thoughts and if I need to clarify anything. I think this proposal makes things flow better and less confusing for the end user on how they should contribute.

Comment (cbhernan) [reply CONTRIBUTING.md]: Sure. I've made the update!

**Review by jaylenw — APPROVED** [review]

**Review by nickchaula — APPROVED** [review]
LGTM. TY DevOps for reviewing. Approved

**Pull request merged**
